index.html

```diff
@@ -38,5 +38,106 @@ <h1>I'm Shimels, Glad to see you!</h1>
+          euismod nisi, ac bibendum odio bibendum vel. Donec euismod nisl ut
+          magna aliquam, vel lacinia sapien bibendum. Sed eget velit vel velit
+          lacinia dapibus.
+        </p>
+        <ul class="tools">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+
+        <button class="see-project-btn">Show Project</button>
+      </div>
+      <div class="card">
+        <img src="images/bk-image.jpg" alt="project image" />
+        <h2>Tonic</h2>
+        <h4>
+          <span>CANOPY</span> &nbsp; ● &nbsp; Back End Dev &nbsp; ● &nbsp; 2015
+        </h4>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed euismod
+          euismod nisi, ac bibendum odio bibendum vel. Donec euismod nisl ut
+          magna aliquam, vel lacinia sapien bibendum. Sed eget velit vel velit
+          lacinia dapibus.
+        </p>
+        <ul class="tools">
+          <li>HTML</li>
+          <li>CSS</li>
+          <li>JavaScript</li>
+        </ul>
+
+        <button class="see-project-btn">Show Project</button>
+      </div>
+    </main>
+    <main id="about-me">
+      <h1>I'm Shimels, Glad to see you!</h1>
+      <p>
+        Full-stack developer with over 3 years of experience in developing web
+        applications. Proficient in both front-end and back-end technologies,
+        including Angular, ASP.NET Core Web API, Node.js, Flask, and SQL.
+        Skilled in troubleshooting and maintaining databases and server-side
+        infrastructure. Adept at analytical thinking and problem-solving.
+        Excellent communication skills and customer focus.
+      </p>
+      <span>Let's Connect</span>
+      <section id="social-links">
+        <a href="#"> <img src="icones/twitter.png" alt="" /></a>
+        <a href="#"> <img src="icones/linkedin.png" alt="" /></a>
+        <a href="#"> <img src="icones/M.png" alt="" /></a>
+        <a href="#"> <img src="icones/github.png" alt="" /></a>
+        <a href="#"> <img src="icones/Vector.png" alt="" /></a>
+      </section>
+
+      <button class="see-project-btn">Get my resume</button>
+
+      <ul>
+        <li>
+          <span class="toggle"><span class="text">Languages</span><span><img src="icones/down.png"></span> </span>
+          <ul class="content">
+            <li><span><img src="icones/js.png"></span> <span class="text">JavaScript</span></li>
+            <li><span><img src="icones/html5.png"></span> <span class="text">HTML</span></li>
+            <li><span><img src="icones/css.png"></span> <span class="text">CSS</span></li>
+          </ul>
+        </li>
+        <li>
+          <span class="toggle"><span class="text">Frameworks</span><span><img src="icones/down.png"></span></span>
+        </li>
+        <li>
+          <span class="toggle"><span class="text">Skills</span><span><img src="icones/down.png"></span></span>
+        </li>
+      </ul>
+    </main>
```

Comment:
- [ ] Nice work implementing the "about-me" section, it looks really good :clap: However it would be nice to wrap it with a `section` tag for semantics reasons.